app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -111,6 +164,45 @@ export const ListingDetailsEligibility = ({
         <StandardTable headers={occupancyTableHeaders} data={occupancyTableData} />
       </ListSection>
       <ListingDetailsPreferences listingID={listing.listingID} />
+      {priorityUnits?.length > 0 ? (
+        <ListSection
+          title={t("listings.priorityUnits")}
+          subtitle={t("listings.priorityUnitsDescription")}
+        >
+          {priorityUnits
+            .filter((unit: ReducedUnit) => {
+              return unit?.name !== "Adaptable"
+            })
+            .map((unit: ReducedUnit) => {
+              return (
+                <InfoCard
+                  title={defaultIfNotTranslated(
+                    priorityLabelMap[unit?.name]?.titleTranslation,
+                    unit.name
+                  )}
+                  subtitle={
+                    unit.numberOfUnits === 1
+                      ? `${unit.numberOfUnits} ${defaultIfNotTranslated(
+                          "listings.features.unit",
+                          "unit"
+                        )}`
+                      : `${unit.numberOfUnits} ${defaultIfNotTranslated("t.units", "units")}`
+                  }
+                >
+                  {defaultIfNotTranslated(
+                    "listings.unitsHaveAccessibilityFeaturesFor",
+                    `These units have accessibility features for people with ${unit.name}.`,
+                    {
+                      type: priorityLabelMap[unit?.name]?.description || unit.name,
```

Comment:
Looks like we're carrying over translation concatenation logic from angular which is not good practice. Can we flatten out the text props which will allow for proper translations? e.g.

```
"listings.unitsHaveAccessibilityFeaturesFor.impaired mobility":  "These units have accessibility features for people with impaired mobility.",
"listings.unitsHaveAccessibilityFeaturesFor.impaired vision and/or hearing": "These units have accessibility features for people with impaired vision and/or hearing."
```

I'm also inclined to leave the default description blank.

Reply:
I was thinking we'd only leave it in until we get the new translations. I'm happy to add those the flattened versions in! I considered it, but was worried about the non english translations - is it ok if those are fully english until we get the proper translations? If we add those fields to the en.json file, then the non-english translations will default to those values. By leave the default description blank - meaning it'll just render an empty string if it can't find a translation?

Reply:
Yes, that is our current process. For any new text properties, they will display only in english until we process our next batch of translations. We should note it in the review instructions to avoid confusion (which you already did, thanks!) and then we should also make sure to add it to our 'next batch of translations' doc.

I hesitate to add any programmatic string concatenation, but i guess if it's just for the default/missing scenario and english only it can be ok in this case.